package.json

```diff
@@ -20,19 +33,22 @@
 		"redis": "^2.8.0",
 		"request": "^2",
 		"serverless-aws-static-file-handler": "^1.0.0",
-		"snowplow-tracker": "^0.3.0"
+		"snowplow-tracker": "^0.3.0",
+		"source-map-support": "^0.5.13"
 	},
 	"scripts": {
+		"lint": "standard",
 		"test": "jest --forceExit --coverage"
 	},
+	"standard": {
+		"env": {
+			"node": true,
+			"jest": true
+		}
+	},
 	"jest": {
-		"verbose": true,
-		"testEnvironment": "node",
-		"setupTestFrameworkScriptFile": "./test/test-setup.js",
-		"coveragePathIgnorePatterns": [
```

Comment:
Are these ignores no longer needed?

Reply:
All of that's handled by `babel-jest` now as it needs to transform files before jest can run them.